pkgs/tools/nix/help/help.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -eu
+
+if [ "${1:-}" = "--debug" ]; then
+    set -x
+fi
+
+nixev() {
+    nix-instantiate --eval --strict -E "$1"
+}
+
+desc_system() {
+    nixev '(import <nixpkgs> {}).system'
+}
+
+desc_multi_user() {
+    if test -e /nix/var/nix/daemon-socket; then
```

Comment:
I just realised that this will probably always exist on darwin, unless nix was installed before the multi-user installer.

Reply:
I feel okay with this, but if there is a good fix / improvement I'd be happy to do that

Reply:
What about also checking `test $NIX_REMOTE = daemon` and printing unknown instead if the results are inconsistent